Source/control.h

```diff
@@ -27,8 +27,10 @@ namespace devilution {
 
 #define SPANEL_WIDTH 320
 #define SPANEL_HEIGHT 352
-#define RIGHT_PANEL (gnScreenWidth - SPANEL_WIDTH)
-#define RIGHT_PANEL_X RIGHT_PANEL
+#define LEFT_PANEL_X 0
+#define LEFT_PANEL_Y ((gnScreenHeight - PANEL_HEIGHT - SPANEL_HEIGHT) / 2)
+#define RIGHT_PANEL_X (gnScreenWidth - SPANEL_WIDTH)
+#define RIGHT_PANEL_Y LEFT_PANEL_Y

```

Comment:
When changing this to be configurable these should probably be replaced with a function that returns a point when given an enum that represents one of the 4 panels.

`Point GetPanelPosition(UiPanels panel);`

Reply:
I will do the refactoring for this.